packages/cli/src/extensions/domains-extension.ts

```diff
@@ -48,7 +48,7 @@ module.exports = (toolbox: GluegunToolbox) => {
       }
     );
 
-    return fetchDomainsResponse.data.data;
+    return fetchDomainsResponse?.data?.data;
   };
 
   toolbox.domains.prepareDomainsMap = (domains, pwaHost: string) => {
```

Comment:
@elkmod what about the domains starting with a hostname other than the passed `pwaHost`. maybe we should skip them and not include in the list of available domains? I think there will be a problem with the routing because our `availableDomains` would have a URLS with foreign domains - not matching the pwa's hostname.

Reply:
Yes, makes sense. We currently don't store any `pwa` flag for the domains. Let's just match `pwaHost` against the `domain.url` and remove the remaining domains.

Reply:
done :heavy_check_mark: